app/views/users/_audit_summary_query_builder.html.erb

```diff
@@ -0,0 +1,3 @@
+<div id="audit_summary_query_builder" class="input-group" style="display: none;">
+  <%= text_f(f, :mail_query, {:label => _('Query for %{mailer}') % {:mailer => mailer.humanize }, :id => 'search',:class => "autocomplete-input", :data => {:url => auto_complete_search_audits_path} }) %>
```

Comment:
Unsure if this is your bug or not, but I see two clear icons:

![screenshot from 2015-05-13 14 35 38](https://cloud.githubusercontent.com/assets/482089/7611605/6062f17a-f97d-11e4-9849-331cf84236f6.png)